### PR TITLE
[SCB-1054]when download file, we should ignore consumer acceptType

### PR DESCRIPTION
**Pull request by heyile**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [ ] Each commit in the pull request should have a meaningful subject line and body.
 - [ ] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [ ] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: common/common-rest/src/main/java/org/apache/servicecomb/common/rest/definition/RestOperationMeta.java [hunk 1]
```diff
@@ -105,6 +110,16 @@ public void init(OperationMeta operationMeta) {
     setAbsolutePath(concatPath(swagger.getBasePath(), operationMeta.getOperationPath()));
   }
 
+  private void checkDownloadFile(Operation operation) {
+    try {
+      Response response = operation.getResponses().get("200");
+      downloadFile = response.getSchema().getType().toLowerCase().equals("file");
+    } catch (Exception e) {
```
Comment (liubao68) [inline common/common-rest/src/main/java/org/apache/servicecomb/common/rest/definition/RestOperationMeta.java]: Do not use exception to do flow control. Check null is better.

File: common/common-rest/src/main/java/org/apache/servicecomb/common/rest/definition/RestOperationMeta.java [hunk 2]
```diff
@@ -217,9 +232,14 @@ public ProduceProcessor ensureFindProduceProcessor(String acceptType) {
     if (StringUtils.isEmpty(acceptType)) {
       return defaultProcessor;
     }
-
-    List<String> mimeTyps = MimeTypesUtils.getSortedAcceptableMimeTypes(acceptType.toLowerCase(Locale.US));
-    for (String mime : mimeTyps) {
+    if (downloadFile) {
```
Comment (liubao68) [inline common/common-rest/src/main/java/org/apache/servicecomb/common/rest/definition/RestOperationMeta.java]: Is it better to add it before StringUtils.isEmpty(acceptType)?

Comment (heyile) [reply common/common-rest/src/main/java/org/apache/servicecomb/common/rest/definition/RestOperationMeta.java]: ok, I will change it. actually, it doesn't matter where it's .
As   **produceProcessorMap.putIfAbsent(MediaType.WILDCARD, defaultProcessor);**
so **defaultProcessor**  equals  **produceProcessorMap.get(MediaType.WILDCARD);**

Comment (heyile) [reply common/common-rest/src/main/java/org/apache/servicecomb/common/rest/definition/RestOperationMeta.java]: ok

File: common/common-rest/src/main/java/org/apache/servicecomb/common/rest/definition/RestOperationMeta.java (hunk 1 above)
Comment (heyile) [reply common/common-rest/src/main/java/org/apache/servicecomb/common/rest/definition/RestOperationMeta.java]: ok

File: common/common-rest/src/main/java/org/apache/servicecomb/common/rest/definition/RestOperationMeta.java [hunk 3]
```diff
@@ -105,6 +111,24 @@ public void init(OperationMeta operationMeta) {
     setAbsolutePath(concatPath(swagger.getBasePath(), operationMeta.getOperationPath()));
   }
 
+  private void checkDownloadFile(Operation operation) {
```
Comment (wujimin) [inline common/common-rest/src/main/java/org/apache/servicecomb/common/rest/definition/RestOperationMeta.java]: 1.change name to checkDownloadFileFlag
2.OperationMeta already collect ResponseMetas, just invoke org.apache.servicecomb.swagger.invocation.response.ResponsesMeta#findResponseMeta, and determined by org.apache.servicecomb.swagger.invocation.response.ResponseMeta#javaType

**Pull request merged**